sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -36,14 +36,6 @@ Interaction with the service using the client library begins with a [client](#te
 To create a client object, you will need the Cognitive Services or Language service `endpoint` to
 your resource and a `credential` that allows you access:
 
-```python
-from azure.core.credentials import AzureKeyCredential
-from azure.ai.textanalytics import TextAnalyticsClient
-
-credential = AzureKeyCredential("<api_key>")
-text_analytics_client = TextAnalyticsClient(endpoint="https://<resource-name>.cognitiveservices.azure.com/", credential=credential)
-```
-
```

Comment:
Apologies, I said the wrong thing. Can we revert this to leave it as it is in main? Otherwise the comment below lost its context. Otherwise LGTM